docker_checker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM rocker/r-devel
+MAINTAINER karl.forner@quartzbio.com
+
+RUN Rscript -e 'install.packages("digest")'
+RUN Rscript -e 'install.packages("crayon")'
+RUN apt-get install -y libcurl4-openssl-dev sudo nano
+
+RUN Rscript -e 'install.packages("devtools")'
+RUN Rscript -e 'install.packages("roxygen2")'
+RUN Rscript -e 'install.packages("testthat")'
+RUN Rscript -e 'devtools::install_github("jimhester/rex")'
```

Comment:
`jimhester/rex` is not the main repository, this should be `kevinushey/rex`.

Also is there a rex feature needed in `covr` that is not available on the CRAN version of `rex`? (I do not recall one offhand)

If there is I should do a new `rex` release to CRAN.